spec/services/data_requests/write_user_events_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe DataRequests::WriteUserEvents do
+  describe '#call' do
+    it 'writes a file with event information' do
+      user_report = JSON.parse(
+        File.read('spec/fixtures/data_request.json'), symbolize_names: true
+      )[0]
```

Comment:
nit, I prefer `.first` to brackets when possible

```suggestion
      ).first
```